code/game/objects/structures/inflatable.dm

```diff
@@ -0,0 +1,238 @@
+/obj/item/inflatable
+	name = "inflatable"
+	w_class = W_CLASS_MEDIUM
+	icon = 'icons/obj/inflatable.dmi'
+	w_type = RECYK_METAL
+	melt_temperature = MELTPOINT_PLASTIC
+	starting_materials = list(MAT_PLASTIC = 1.5*CC_PER_SHEET_MISC)
+
+	var/deploy_path = null
+
+/obj/item/inflatable/attack_self(mob/user)
+	if(!deploy_path)
+		return
+	if(!istype(user.loc, /turf))
+		return
+
+	if(user.drop_item(src))
```

Comment:
You can control click them as a shortcut, because verbs are bad.
Also: compare this to dropping fulltile reinforced windows which also have no build delay and fit 25 in one stack of glass.